### PR TITLE
update deployment docs

### PR DESCRIPTION
**Pull request by disksing**
Signed-off-by: disksing <i@disksing.com>

### What is changed, added or deleted? (Required)
Update deployment docs. Replace term `城` with `区域`, replace `中心` with `AZ`.

Pictures are WIP.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by disksing** [comment]
images LGTM!

File: two-data-centers-in-one-city-deployment.md [hunk 1]
```diff
@@ -82,7 +84,7 @@ alertmanager_servers:
 
 ### Placement Rules 规划
 
-为了按照规划的集群拓扑进行部署，你需要使用 [Placement Rules](/configure-placement-rules.md) 来规划集群副本的放置位置。以 4 副本（2 Voter 副本在主数据中心，1 Voter 和 1 Learner 在从数据中心）的部署方式为例，可使用 Placement Rules 进行如下副本配置：
+为了按照规划的集群拓扑进行部署，你需要使用 [Placement Rules](/configure-placement-rules.md) 来规划集群副本的放置位置。以 4 副本（2 Voter 副本在主 AZ，1 Voter 和 1 Learner 在从 AZ）的部署方式为例，可使用 Placement Rules 进行如下副本配置：
```
Comment (shichun-0415) [inline two-data-centers-in-one-city-deployment.md:87]:
```suggestion
为了按照规划的集群拓扑进行部署，你需要使用 [Placement Rules](/configure-placement-rules.md) 来规划集群副本的放置位置。以 4 副本（2 个 Voter 副本在主 AZ，1 个 Voter 副本和 1 个 Learner 副本在从 AZ）的部署方式为例，可使用 Placement Rules 进行如下副本配置：
```

File: two-data-centers-in-one-city-deployment.md [hunk 2]
```diff
@@ -232,9 +234,9 @@ cat default.json
 配置项说明：
 
 + `replication-mode` 为待启用的复制模式，以上示例中设置为 `dr-auto-sync`。默认使用 majority 算法。
-+ `label-key` 用于区分不同的数据中心，需要和 Placement Rules 相匹配。其中主数据中心为 "east"，从数据中心为 "west"。
-+ `primary-replicas` 是在主数据中心 Voter 副本的数量。
-+ `dr-replicas` 是在从数据中心 Voter 副本的数量。
++ `label-key` 用于区分不同的 AZ，需要和 Placement Rules 相匹配。其中主 AZ 为 "east"，从 AZ 为 "west"。
++ `primary-replicas` 是在主 AZ Voter 副本的数量。
++ `dr-replicas` 是在从 AZ Voter 副本的数量。
```
Comment (shichun-0415) [inline two-data-centers-in-one-city-deployment.md:239]:
```suggestion
+ `primary-replicas` 是主 AZ 上 Voter 副本的数量。
+ `dr-replicas` 是从 AZ 上 Voter 副本的数量。
```

File: two-data-centers-in-one-city-deployment.md [hunk 3]
```diff
@@ -261,27 +263,27 @@ curl http://pd_ip:pd_port/pd/api/v1/replication_mode/status
 
 简单来讲，集群的复制模式可以自动在三种状态之间自适应的切换：
 
-- 当集群一切正常时，会进入同步复制模式来最大化地保障灾备机房的数据完整性。
-- 当机房网络断连或灾备机房发生整体故障时，在经过一段提前设置好的保护窗口之后，集群会进入异步复制状态，来保障业务的可用性。
-- 当机房网络重连或灾备机房整体恢复之后，灾备机房的 TiKV 节点会重新加入到集群，逐步同步数据并最终转为同步复制模式。
+- 当集群一切正常时，会进入同步复制模式来最大化地保障灾备 AZ 的数据完整性。
+- 当 AZ 网络断连或灾备 AZ 发生整体故障时，在经过一段提前设置好的保护窗口之后，集群会进入异步复制状态，来保障业务的可用性。
+- 当 AZ 网络重连或灾备 AZ 整体恢复之后，灾备 AZ 的 TiKV 节点会重新加入到集群，逐步同步数据并最终转为同步复制模式。
```
Comment (shichun-0415) [inline two-data-centers-in-one-city-deployment.md:268]: 灾备 AZ 是指从 AZ 吧
```suggestion
- 当集群一切正常时，会进入同步复制模式来最大化地保障灾备 AZ 的数据完整性。
- 当 AZ 网络断连或灾备 AZ 发生整体故障时，在经过一段提前设置好的保护窗口之后，集群会进入异步复制状态，来保障业务的可用性。
- 当 AZ 网络重连或灾备 AZ 整体恢复之后，灾备 AZ 的 TiKV 节点会重新加入到集群，逐步同步数据并最终转为同步复制模式。
```

File: two-data-centers-in-one-city-deployment.md [hunk 4]
```diff
@@ -261,27 +263,27 @@ curl http://pd_ip:pd_port/pd/api/v1/replication_mode/status
...
 
 状态转换的细节过程如下：
 
 1. **初始化**：集群在初次启动时处于 sync（同步复制）模式，PD 会下发信息给 TiKV，所有 TiKV 节点会严格按照 sync 模式的要求进行工作。
 
-2. **同步切异步**：PD 通过定时检查 TiKV 的心跳信息来判断 TiKV 是否宕机或断连。如果宕机数超过 PRIMARY/DR 各自副本的数量 `primary-replicas` 和 `dr-replicas`，意味着无法完成同步复制，需要切换状态。当宕机时间超过了 `wait-store-timeout` 设定的时间，PD 将集群状态切换成 async（异步复制）模式。然后 PD 再将 async 状态下发到所有 TiKV 节点，TiKV 的复制模式由双中心同步方式转为原生的 Raft 大多数落实方式 (majority)。
+2. **同步切异步**：PD 通过定时检查 TiKV 的心跳信息来判断 TiKV 是否宕机或断连。如果宕机数超过 PRIMARY/DR 各自副本的数量 `primary-replicas` 和 `dr-replicas`，意味着无法完成同步复制，需要切换状态。当宕机时间超过了 `wait-store-timeout` 设定的时间，PD 将集群状态切换成 async（异步复制）模式。然后 PD 再将 async 状态下发到所有 TiKV 节点，TiKV 的复制模式由双 AZ 同步方式转为原生的 Raft 大多数落实方式 (majority)。
```
Comment (shichun-0415) [inline two-data-centers-in-one-city-deployment.md:274]:
```suggestion
2. **同步切异步**：PD 通过定时检查 TiKV 的心跳信息来判断 TiKV 是否宕机或断连。如果宕机数超过 Primary 和 DR 各自副本的数量 `primary-replicas` 和 `dr-replicas`，意味着无法完成同步复制，需要切换状态。当宕机时间超过了 `wait-store-timeout` 设定的时间，PD 将集群状态切换成 async（异步复制）模式。然后 PD 再将 async 状态下发到所有 TiKV 节点，TiKV 的复制模式由双 AZ 同步方式转为原生的 Raft 大多数落实方式 (majority)。
```

File: two-data-centers-in-one-city-deployment.md [hunk 5]
```diff
@@ -261,27 +263,27 @@ curl http://pd_ip:pd_port/pd/api/v1/replication_mode/status
...
 
-3. **异步切同步**：PD 通过定时检查 TiKV 的心跳信息来判断 TiKV 是否恢复连接，如果宕机数小于 PRIMARY/DR 各自副本的数量，意味着可以切回同步了。PD 会将集群复制状态先切换至 sync-recover，再将该状态下发给所有 TiKV 节点。TiKV 的所有 Region 逐步切换成双机房同步复制模式，切换成功后通过心跳将状态同步信息给 PD。PD 记录 TiKV 上 Region 的状态并统计恢复进度。当 TiKV 的所有 Region 都完成了同步复制模式的切换，PD 将集群复制状态切换为 sync。
+3. **异步切同步**：PD 通过定时检查 TiKV 的心跳信息来判断 TiKV 是否恢复连接，如果宕机数小于 PRIMARY/DR 各自副本的数量，意味着可以切回同步了。PD 会将集群复制状态先切换至 sync-recover，再将该状态下发给所有 TiKV 节点。TiKV 的所有 Region 逐步切换成双 AZ 同步复制模式，切换成功后通过心跳将状态同步信息给 PD。PD 记录 TiKV 上 Region 的状态并统计恢复进度。当 TiKV 的所有 Region 都完成了同步复制模式的切换，PD 将集群复制状态切换为 sync。
```
Comment (shichun-0415) [inline two-data-centers-in-one-city-deployment.md:276]:
```suggestion
3. **异步切同步**：PD 通过定时检查 TiKV 的心跳信息来判断 TiKV 是否恢复连接，如果宕机数小于 Primary 和 DR 各自副本的数量，意味着可以切回同步。PD 会将集群复制状态先切换至 sync-recover，再将该状态下发给所有 TiKV 节点。TiKV 的所有 Region 逐步切换成双 AZ 同步复制模式，切换成功后通过心跳将状态同步信息给 PD。PD 记录 TiKV 上 Region 的状态并统计恢复进度。当 TiKV 的所有 Region 都完成了同步复制模式的切换，PD 将集群复制状态切换为 sync。
```

File: multi-data-centers-in-one-city-deployment.md [hunk 6]
```diff
@@ -1,12 +1,14 @@
 ---
-title: 同城多数据中心部署 TiDB
-summary: 本文档介绍同城多数据中心部署 TiDB 方案。
+title: 单区域多 AZ 部署 TiDB
+summary: 本文档介绍单区域多 AZ 部署 TiDB 方案。
 aliases: ['/docs-cn/dev/multi-data-centers-in-one-city-deployment/','/docs-cn/dev/how-to/deploy/geographic-redundancy/overview/','/docs-cn/dev/geo-redundancy-deployment/']
```
Comment (shichun-0415) [inline multi-data-centers-in-one-city-deployment.md:4]:
```suggestion
aliases: ['/docs-cn/dev/how-to/deploy/geographic-redundancy/overview/','/docs-cn/dev/geo-redundancy-deployment/']

```

File: multi-data-centers-in-one-city-deployment.md [hunk 7]
```diff
@@ -19,49 +21,49 @@ Raft 是一种分布式一致性算法，在 TiDB 集群的多种组件中，PD
...
-![三中心部署读性能优化](/media/deploy-3dc-optimize.png)
+![三 AZ 部署读性能优化](/media/deploy-3dc-optimize.png)
 
 **优点：**
 
 集群 TSO 获取能力以及读取性能有所提升。具体调度策略设置模板参照如下：
 
 ```shell
--- 其他机房统一驱逐 leader 到业务流量机房
+-- 其他 AZ 统一驱逐 leader 到业务流量 AZ
```
[the remark this answers is not in the document]
Comment (disksing) [reply multi-data-centers-in-one-city-deployment.md:79]: 是这样的

File: multi-data-centers-in-one-city-deployment.md [hunk 8]
```diff
@@ -77,18 +79,18 @@ member leader_priority pdName3 3
...
 **缺点：**
 
-- 写入场景仍受数据中心网络延迟影响，这是因为遵循 Raft 多数派协议，所有写入的数据需要同步复制到至少 2 个数据中心
-- TiDB Server 数据中心级别单点
-- 业务流量纯走单数据中心，性能受限于单数据中心网络带宽压力
-- TSO 获取能力以及读取性能受限于业务流量数据中心集群 PD、TiKV 组件是否正常，否则仍受跨数据中心网络交互影响
+- 写入场景仍受 AZ 网络延迟影响，这是因为遵循 Raft 多数派协议，所有写入的数据需要同步复制到至少 2 个 AZ
+- TiDB Server 是 AZ 级别单点
+- 业务流量纯走单 AZ，性能受限于单 AZ 网络带宽压力
+- TSO 获取能力以及读取性能受限于业务流量 AZ 集群 PD、TiKV 组件是否正常，否则仍受跨 AZ 网络交互影响
```
[the remark this answers is not in the document]
Comment (disksing) [reply multi-data-centers-in-one-city-deployment.md:99]: 应该是说如果有节点故障的情况，Region leader会在别的AZ选出来，这样的话前面说的控制leader位置的机制就失效了。

File: two-data-centers-in-one-city-deployment.md [hunk 9]
```diff
@@ -1,41 +1,43 @@
...
 
-本文介绍同城两中心的部署模式，相关架构、示例配置、副本方法、启用该模式的方法。
+本文介绍单区域双 AZ 的部署模式，相关架构、示例配置、副本方法、启用该模式的方法。
+
+本文中的区域指的是地理隔离的不同位置，AZ（可用区）指的是区域内部划分的相互独立的资源集合，这里所描述的方案同样适用于同一城市两个数据中心（同城双中心）的场景。
 
 ## 简介
 
-TiDB 在 on-premises 部署的场景下，通常采用多中心部署方案，以保证高可用和容灾能力。多中心部署方案包括两地三中心部署模式、同城三中心部署模式等多种部署模式。本文介绍同城两中心部署方案，即在一座城市部署两个数据中心，成本更低，同样能满足高可用和容灾要求。该部署方案采用自适应同步模式，即 Data Replication Auto Synchronous，简称 DR Auto-Sync。
+TiDB 在 on-premises 部署的场景下，通常采用多 AZ 部署方案，以保证高可用和容灾能力。多 AZ 部署方案包括单区域多 AZ 部署模式、双区域多 AZ 部署模式等多种部署模式。本文介绍单区域双 AZ 部署方案，即在同一区域部署两个 AZ，成本更低，同样能满足高可用和容灾要求。该部署方案采用自适应同步模式，即 Data Replication Auto Synchronous，简称 DR Auto-Sync。
```
[the remark this answers is not in the document]
Comment (disksing) [reply two-data-centers-in-one-city-deployment.md:14]: 有道理，要不这一段还是改回原来的city/dc的说法？

File: two-data-centers-in-one-city-deployment.md (hunk 3 above)
Comment (disksing) [reply two-data-centers-in-one-city-deployment.md:267]: 对

File: two-data-centers-in-one-city-deployment.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (disksing) [reply two-data-centers-in-one-city-deployment.md:14]: 好像改回去也不合适，这段属于整个文档的简介，要不就直接把 on-premises 的说法去掉吧？

[the remark this answers is not in the document]
Comment (shichun-0415) [reply two-data-centers-in-one-city-deployment.md:14]: 去掉 on-premise 我觉得可以。
> TiDB 通常采用多 AZ 部署方案保证集群高可用和容灾能力。

File: multi-data-centers-in-one-city-deployment.md [hunk 10]
```diff
@@ -1,12 +1,16 @@
 ---
-title: 同城多数据中心部署 TiDB
-summary: 本文档介绍同城多数据中心部署 TiDB 方案。
-aliases: ['/docs-cn/dev/multi-data-centers-in-one-city-deployment/','/docs-cn/dev/how-to/deploy/geographic-redundancy/overview/','/docs-cn/dev/geo-redundancy-deployment/']
+title: 单区域多 AZ 部署 TiDB
+summary: 本文档介绍单个区域多个可用区部署 TiDB 的方案。
+
+aliases: ['/docs-cn/dev/how-to/deploy/geographic-redundancy/overview/','/docs-cn/dev/geo-redundancy-deployment/']
```
Comment (shichun-0415) [inline multi-data-centers-in-one-city-deployment.md:5]:
```suggestion
summary: 本文档介绍单个区域多个可用区部署 TiDB 的方案。
aliases: ['/docs-cn/dev/how-to/deploy/geographic-redundancy/overview/','/docs-cn/dev/geo-redundancy-deployment/']

```

File: multi-data-centers-in-one-city-deployment.md [hunk 11]
```diff
@@ -1,12 +1,16 @@
 ---
-title: 同城多数据中心部署 TiDB
-summary: 本文档介绍同城多数据中心部署 TiDB 方案。
-aliases: ['/docs-cn/dev/multi-data-centers-in-one-city-deployment/','/docs-cn/dev/how-to/deploy/geographic-redundancy/overview/','/docs-cn/dev/geo-redundancy-deployment/']
+title: 单区域多 AZ 部署 TiDB
+summary: 本文档介绍单个区域多个可用区部署 TiDB 的方案。
+aliases: ['/docs-cn/dev/how-to/deploy/geographic-redundancy/overview/','/docs-cn/dev/geo-redundancy-deployment/']
+
```
Comment (shichun-0415) [inline multi-data-centers-in-one-city-deployment.md:5]:
```suggestion
summary: 本文档介绍单个区域多个可用区部署 TiDB 的方案。
aliases: ['/docs-cn/dev/how-to/deploy/geographic-redundancy/overview/','/docs-cn/dev/geo-redundancy-deployment/']

```

File: two-data-centers-in-one-city-deployment.md [hunk 12]
```diff
@@ -1,41 +1,42 @@
...
 
-本文介绍同城两中心的部署模式，相关架构、示例配置、副本方法、启用该模式的方法。
+本文介绍单区域双可用区 (Availability Zone, AZ) 的部署模式，包括方案架构、示例配置、副本方法及启用该模式的方法。
+
+本文中的区域指的是地理隔离的不同位置，AZ 指的是区域内部划分的相互独立的资源集合，本文所描述的方案同样适用于同一城市两个数据中心（同城双中心）的场景。
 
 ## 简介
 
-TiDB 在 on-premises 部署的场景下，通常采用多中心部署方案，以保证高可用和容灾能力。多中心部署方案包括两地三中心部署模式、同城三中心部署模式等多种部署模式。本文介绍同城两中心部署方案，即在一座城市部署两个数据中心，成本更低，同样能满足高可用和容灾要求。该部署方案采用自适应同步模式，即 Data Replication Auto Synchronous，简称 DR Auto-Sync。
+TiDB 在 on-premises 部署的场景下，通常采用多 AZ 部署方案，以保证高可用和容灾能力。多 AZ 部署方案包括单区域多 AZ 部署模式、双区域多 AZ 部署模式等多种部署模式。本文介绍单区域双 AZ 部署方案，即在同一区域部署两个 AZ，成本更低，同样能满足高可用和容灾要求。该部署方案采用自适应同步模式，即 Data Replication Auto Synchronous，简称 DR Auto-Sync。
```
Comment (shichun-0415) [inline two-data-centers-in-one-city-deployment.md:14]:
```suggestion
TiDB 通常采用多 AZ 部署方案保证集群高可用和容灾能力。多 AZ 部署方案包括单区域多 AZ 部署模式、双区域多 AZ 部署模式等多种部署模式。本文介绍单区域双 AZ 部署方案，即在同一区域部署两个 AZ，成本更低，同样能满足高可用和容灾要求。该部署方案采用自适应同步模式，即 Data Replication Auto Synchronous，简称 DR Auto-Sync。
```

File: multi-data-centers-in-one-city-deployment.md [hunk 13]
```diff
@@ -19,49 +21,51 @@ Raft 是一种分布式一致性算法，在 TiDB 集群的多种组件中，PD
...
-![三中心部署读性能优化](/media/deploy-3dc-optimize.png)
+![三 AZ 部署读性能优化](/media/deploy-3dc-optimize.png)
 
 **优点：**
 
 集群 TSO 获取能力以及读取性能有所提升。具体调度策略设置模板参照如下：
 
 ```shell
--- 其他机房统一驱逐 leader 到业务流量机房
+-- 其他 AZ 统一驱逐 leader 到业务流量 AZ
```
Comment (shichun-0415) [inline multi-data-centers-in-one-city-deployment.md:68]:
```suggestion
-- 其他 AZ 将 leader 驱逐至承载业务流量的 AZ

```

File: two-data-centers-in-one-city-deployment.md [hunk 14]
```diff
@@ -1,41 +1,42 @@
...
 
-同城两中心部署方案下，两个数据中心距离在 50 km 以内，通常位于同一个城市或两个相邻城市（例如北京和廊坊），数据中心间的网络连接延迟小于 1.5 ms，带宽大于 10 Gbps。
+单区域双 AZ 部署方案下，两个 AZ 通常位于同一个城市或两个相邻城市（例如北京和廊坊）， 相距 50 km 以内，AZ 间的网络连接延迟小于 1.5 ms，带宽大于 10 Gbps。
 
 ## 部署架构
 
-本文以某城市为例，城里有两个数据中心 IDC1 和 IDC2，分为位于城东和城西。
+本文以某市为例，市里有两个 AZ AZ1 和 AZ2，AZ1 为主用区域，AZ2 为从属区域，分别位于城东和城西。
```
Comment (shichun-0415) [inline two-data-centers-in-one-city-deployment.md:20]:
```suggestion
本文以某市为例，市里有两个 AZ，AZ1 和 AZ2，AZ1 为主用区域，AZ2 为从属区域，分别位于城东和城西。
```

File: three-data-centers-in-two-cities-deployment.md [hunk 15]
```diff
@@ -1,57 +1,61 @@
 ---
-title: 两地三中心部署
-summary: 介绍两地三中心部署方式。
-aliases: ['/docs-cn/dev/three-data-centers-in-two-cities-deployment/']
+title: 双区域多 AZ 部署
```
Comment (shichun-0415) [inline three-data-centers-in-two-cities-deployment.md:2]:
```suggestion
title: 双区域多 AZ 部署 TiDB
```

File: three-data-centers-in-two-cities-deployment.md [hunk 16]
```diff
@@ -1,57 +1,61 @@
...
+summary: 介绍在两个区域多个可用区部署 TiDB 的方式。
 ---
 
-# 两地三中心部署
+# 双区域多 AZ 部署
```
Comment (shichun-0415) [inline three-data-centers-in-two-cities-deployment.md:6]:
```suggestion
# 双区域多 AZ 部署 TiDB
```

File: two-data-centers-in-one-city-deployment.md [hunk 17]
```diff
@@ -1,41 +1,42 @@
 ---
-title: 同城两中心自适应同步模式部署
-summary: 了解同城使用两中心自适应同步模式部署方式。
+title: 单区域双 AZ 部署
```
Comment (shichun-0415) [inline two-data-centers-in-one-city-deployment.md:2]:
```suggestion
title: 单区域双 AZ 部署 TiDB
```

File: two-data-centers-in-one-city-deployment.md [hunk 18]
```diff
@@ -1,41 +1,42 @@
...
+summary: 了解单个区域两个可用区自适应同步模式部署方式。
 ---
 
-# 同城两中心自适应同步模式部署
+# 单区域双 AZ 自适应同步模式部署
```
Comment (shichun-0415) [inline two-data-centers-in-one-city-deployment.md:6]:
```suggestion
# 单区域双 AZ 部署 TiDB
```

**Pull request closed without merging**